app/models/miq_region_remote.rb

```diff
@@ -92,12 +92,13 @@ def self.with_remote_connection(host, port, username, password, database, adapte
 
     begin
       pool = establish_connection({
-        :adapter  => adapter,
-        :host     => host,
-        :port     => port,
-        :username => username,
-        :password => password,
-        :database => database
+        :adapter         => adapter,
+        :host            => host,
+        :port            => port,
+        :username        => username,
+        :password        => password,
+        :database        => database,
+        :connect_timeout => connect_timeout
```

Comment:
does this work with `ActiveSupport::Duration` objects?  It might be nice to say 
`with_remote_connection(5.seconds)` in `remote_region_lsn`

Reply:
I guess you can always do `connect_timeout.to_i`

Reply:
Yup that works.